device/marvell/arm64-marvell_rd98DX45xx_cn9131-r0/pmon_daemon_control.json

```diff
@@ -0,0 +1,8 @@
+{
+    "skip_ledd": true,
+    "skip_pcied": true,
+    "skip_thermalctld": true,
+    "skip_psud": true,
+    "skip_xcvrd": true,
+    "skip_syseepromd": true
```

Comment:
why all of them are skipped?

Reply:
This is reference design board to verify the asic functionality, hence we are skipping this.

Reply:
@pavannaregundi how do these pmon daemon prevents asic functionlality? or do you see some other issue with these daemon running

Reply:
@prgeor RD boards are not the production version of the boards. We are not supporting pmon functionality in them.